Update metadata for Kai Finance

kklas opened the pull request:
Kai Finance had a small pivot to leveraged yield farming. Update icon and metadata. Add twitter.

realdealshaman commented:
Hey @kklas, thanks for the PR! Has the TVL calculation method also been updated? We have a Leveraged Farming category that can be used, and could you provide a reference link on how Pyth is being used?

kklas commented:
@realdealshaman Yes the TVL calculation is up to date with addition of USDC vault https://github.com/DefiLlama/DefiLlama-Adapters/pull/12162 (we're still using the same vaults). I have updated the category to Leveraged Farming and added a reference to oracle docs (https://docs.kai.finance/vaults/liquidity-provider-vaults/price-oracle). There's more info on how oracles are used in our whitepaper https://docs.kai.finance/resources/whitepaper
